README.md

```diff
@@ -635,10 +635,16 @@ The following command starts the client development server at port 8081:
 make client-dev-server
 ```
 
-Now run Galaxy (on port 8080):
+Now run Galaxy (on port 8080), but we can skip the client build this time, because we will running the dev server proxied against 8080:
```

Comment:
```suggestion
Now run Galaxy (on port 8080), but we can skip the client build this time, because we will be running the dev server proxied against 8080:
```